lib/components/screens/home/home_tab_view.dart

```diff
@@ -34,7 +34,7 @@ class HomeTabView extends StatelessWidget {
           return const SizedBox();
       }
     } else if (entry.isListType) {
-      return HomeListTimeline(listId: entry.id);
+      return HomeListTimeline(listId: entry.id, name: entry.name,);
```

Comment:
Please make sure to format the file :D

Reply:
If you are using vscode you might want to consider using the recommended dart settings (https://dartcode.org/docs/recommended-settings/)
At least for harpy to enable `editor.formatOnSave`

Reply:
sorry, I forgot to run `flutter format .` on the last commit.